clog/config.py

```diff
@@ -123,6 +123,18 @@
     help=("After a timeout occurs, clog won't write to Monk for as many "
           "milliseconds as defined here"))
 
+monk_use_memory_buffer = clog_namespace.get_bool('monk_use_memory_buffer',
+    default=False,
+    help="Add lines to a memory buffer in case of a timeout.")
+
+monk_memory_buffer_max_bytes = clog_namespace.get_int('monk_memory_buffer_max_bytes',
+    default=10 * 1024 * 1024,
+    help="(Approximate) Maximum size of the memory buffer.")
+
+monk_buffering_time_ms = clog_namespace.get_int('monk_buffering_time_ms',
+    default=500,
```

Comment:
The purpose of this is very similar to timeout_backoff, right? I think we could get away by just relying on timeout_backoff. If buffering is enabled, we could just add to buffer then return, otherwise return directly. Do you think it would work?